standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -296,7 +296,8 @@ private void startLocalSegmentStore(int segmentStoreId) throws Exception {
                                          .with(AutoScalerConfig.TOKEN_SIGNING_KEY, "secret")
                                          .with(AutoScalerConfig.AUTH_ENABLED, this.enableAuth)
                                          .with(AutoScalerConfig.TLS_ENABLED, this.enableTls)
-                                         .with(AutoScalerConfig.TLS_CERT_FILE, this.certFile))
+                                         .with(AutoScalerConfig.TLS_CERT_FILE, this.certFile)
+                                         .with(AutoScalerConfig.VALIDATE_HOSTNAME, false))
```

Comment:
Why are we turning off hostname validation? It might be the right thing to do, but seems unrelated to the fix here.

Reply:
This is intended to make provided sample certificates work in standalone mode. The fix made below clears the way for another error which shows up due to failure in hostname verification when using provided certs; This fix resolves that new issue.
 https://github.com/pravega/pravega/blob/2c85e49fcfa36c3e8c713ea2c2ccf7f364a86db4/segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/stat/AutoScaleProcessor.java#L160